tests/README.md

```diff
@@ -0,0 +1,3 @@
+NOTE: `helpers/helpers.js` cannot be moved to the top-level tests directory. For Windows compatibility, we alias `npm test` to `nodeunit tests`, which runs all the .js files in the top-level directory; helpers.js is not a valid nodeunit file, so this would cause tests to fail.
+
+TODO: document testing infrastructure more
```

Comment:
Move this remark to an issue (unless you're planning to address this as part of this PR)?